openstack_plugin_common/__init__.py

```diff
@@ -563,6 +563,11 @@ def _put_client_in_kw(client_name, client_class, kw):
             config = ctx.target.node.properties.get('openstack_config')
     else:
         config = None
+    if 'openstack_config' in kw:
+        if config:
+            config.update(kw['openstack_config'])
```

Comment:
I think the dictionary should be cloned here, so in case the "openstack_config"  node property is also read somewhere else it wouldn't have changed